pkg/api/versions_test.go

```diff
@@ -272,18 +270,53 @@ func TestVersion_IsDeprecatedIn(t *testing.T) {
 	}
 	for _, tt := range tests {
 		deprecatedVersion := &Version{DeprecatedIn: tt.deprecatedIn}
-		got := deprecatedVersion.IsDeprecatedIn(tt.targetVersion)
+		got := deprecatedVersion.isDeprecatedIn(tt.targetVersion)
 		assert.Equal(t, tt.want, got, "test failed: "+tt.name)
 	}
 }
 
-func Test_VersionListIsValid(t *testing.T) {
-	// This test validates that all of the versions in VersionList are valid semVer
-	// it should prevent us from putting bad values in that list in future development
-	for _, version := range VersionList {
-		if version.DeprecatedIn == "" {
-			t.Skip()
-		}
-		assert.True(t, semver.IsValid(version.DeprecatedIn), fmt.Sprintf("version %s is not valid semver", version.DeprecatedIn))
+func TestVersion_IsRemovedIn(t *testing.T) {
+
+	tests := []struct {
+		name          string
+		targetVersion string
+		want          bool
+		removedIn     string
+	}{
+		{
+			name:          "not removed yet 1.15.0",
+			targetVersion: "v1.15.0",
+			removedIn:     "v1.16.0",
+			want:          false,
+		},
+		{
+			name:          "equal values",
+			targetVersion: "v1.16.0",
+			removedIn:     "v1.16.0",
+			want:          true,
+		},
+		{
+			name:          "greater than",
+			targetVersion: "v1.17.0",
+			removedIn:     "v1.16.0",
+			want:          true,
+		},
+		{
+			name:          "Bad semVer",
```

Comment:
super small nit just to match the other names:
`s/B/b/`

Reply:
```suggestion
			name:          "bad semVer",
```